### PR TITLE
Improve AIO development workflows

### PR DESCRIPTION
**Pull request by devversion**
See individual commits

### CODE REVIEW

**Review by josephperrott — APPROVED** [review]
LGTM

Just one thing I wanted to confirm before merge

File: aio/package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "postinstall": "node tools/cli-patches/patch.js",
     "prebuild": "yarn setup",
-    "build": "bazel build //aio:build --config=release",
+    "build": "bazel build //aio:build",
+    "build-prod": "yarn build --release",
```
[the remark this answers is not in the document]
Comment (devversion) [reply aio/package.json:12]: Yes. good catch.

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit ccff7dcb280e2567150b75c9bc97307500df650f.

**Comment by kormide** [comment]
👍

**Pull request closed without merging**